tools/prepare_mapping_v2.py

```
@@ -0,0 +1,199 @@
+            node["assessable"],
+            node["urn"],
+            node.get("ref_id"),
+            node.get("name"),
+            node.get("description"),
+        ])
+
+    # Save the workbook to disk
+    try:
+        wb_output.save(output_file_name)
+    except Exception as e:
+        raise IOError(f"Failed to save Excel file \"{output_file_name}\"\n\t   {e}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)